### PR TITLE
core models completed.

### PR DESCRIPTION
**Pull request by ErfanMoradi2000**
Core app has been installed, models of the app has been created base on designed ERD and the URL has been included.

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Courses(models.Model):
```
Comment (mohammad330a) [inline core/models.py]: Courses -> Course

Comment (mohammad330a) [reply core/models.py]: Please use singular model names.

Comment (ErfanMoradi2000) [reply core/models.py]: Done. the model names are all singular now.

Comment (mohammad330a) [reply core/models.py]: It looks great. thank you.

**Review by mohammad330a — APPROVED** [review]

**Pull request merged**